### DISCUSSION TITLE
Issue with Android and German Language patch

### DISCUSSION THREAD

**Discussion by smalltoe** [exult/exult#369] [General]
I am able to launch the German patched version of Black Gate on Windows Exult 18, however it breaks Exult 18 on Android.
This only applies to the Black Gate - German patch works for Serpent Isle.
Any idea how to make that work?
Many Thanks!

**Comment by DominusExult** [comment]
How does it break?

**Comment by smalltoe** [comment]
Im able to "Launch Exult" with clicking the corresponding button. When I select "Ultima VII The Forge of Virtue" either I get back to Exult main menu or I get an information from Android saying "Exult is repeatedly terminated" and I can chose what to do next.

**Comment by DominusExult** [comment]
hmm, hang in there, maybe I can get snapshots of Exult for Android going this week. It may be that the 1.8 version doesn't like something there.
Have you used the same folders that you used for the Windows version?

**Reply by smalltoe** [reply]
Yes exactly the same - after I had the issue on Android I wanted to crosscheck the same on Windows

**Comment by DominusExult** [comment]
can you give this apk a try? https://www.dropbox.com/scl/fi/e2eu0wrfgw06k85uqts8v/exult-snapshot.apk?rlkey=yllpj8257sm8q41zrafe767pm&dl=0

**Reply by smalltoe** [reply]
Many Thnks, I tried this one with the same result as above (additionally the sound is fragmented)

**Reply by DominusExult** [reply]
Thanks for testing. Bummer it doesn't automagically work with that ;(
Choppy sound might be due to it being a debug build.

Could you make the translated game available for (email address deleted after receiving the game)? (So that I have exactly the same as the one causing problems for you).

**Reply by smalltoe** [reply]
Done, I hope the zip goes through.

**Comment by DominusExult** [comment]
> Many Thnks, I tried this one with the same result as above (additionally the sound is fragmented)

Did it actually install the debug version? I think you need to uninstall the release first before being able to install the debug version. Just checking before diving into this more :)

**Reply by smalltoe** [reply]
Yes, I just doublechcked once more. Behaviour is as described above. Many Thanks for looking into this ;-)

**Comment by ungesundes-halbwissen** [comment]
Hi smalltoe, sorry for not answering any earlier. Glad you found your way to the Exult forums - I would have (had to) pointed you here anyway and Dominus is the perfect one to ask :-)

John

**Comment by DominusExult** [comment]
@smalltoe I'm really sorry, I completely forgot about it, summer holiday other things and as it wasn't in our issues I overlooked it because other discussions pushed it down. I'll look into how to integrate the translated versions onto Android.
